evalml/tests/component_tests/decomposer_tests/test_decomposer.py

```diff
@@ -16,6 +16,10 @@
 decomposer_list = [STLDecomposer, PolynomialDecomposer]
 
 
+def get_trend_dataframe_format_correct(df):
```

Comment:
Just moved this up from the child class test modules.

Reply:
What is this test testing?

Reply:
It's more a utility for the tests to use, just to make sure that `get_trend_df()` returns a dataframe with the proper columns.  `get_trend_dataframe_format_correct()` will be used in the `get_trend_dataframe()` tests.